src/main/java/org/folio/circulation/resources/CheckOutByBarcodeResource.java

```diff
@@ -11,6 +12,12 @@
 
 import java.util.UUID;
 
+import io.vertx.core.http.HttpClient;
```

Comment:
I don't follow what import ordering convention is being followed here.

Please could the provisionally agreed convention be used, as described in https://issues.folio.org/browse/FOLIO-1667